tests/cpptests/test_cpp_index.cpp

```diff
@@ -440,13 +440,13 @@ TEST_F(IndexTest, testNumericInverted) {
     size_t sz = InvertedIndex_WriteNumericEntry(idx, i + 1, (double)(i + 1));
     // printf("written %zd bytes\n", sz);
 
-    ASSERT_TRUE(sz > 1);
+    ASSERT_TRUE(sz > (i ? 1 : 0)); // first doc has zero delta (not written)
```

Comment:
remove

Reply:
Why remove?

Reply:
Won't it be always true?

Reply:
It's part of the test (write something to the inverted index)